packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dr-response.ts

```diff
@@ -189,6 +190,15 @@ async function handleSuccessResponse({
       )
       .map(result => result.value);
 
+    const processedIds = new Set(
+      documentReferences.map(doc => doc.metriportId).filter((id): id is string => id != null)
+    );
+    const operationOutcome = generateOperationOutcomesForMissingDocuments(
+      idMapping,
+      processedIds,
+      outboundRequest.id
+    );
+
```

Comment:
Is this to handle the case where we send 10 requests and for some reason only get back 9 downloaded docs?